vulcan/_data.py

```diff
@@ -58,16 +59,26 @@ async def get_addressbook(
         return Addressbook.get(self._api, **kwargs)
 
     async def get_messages(
-        self, last_sync: datetime = None, folder=1, **kwargs
+        self, message_box, last_sync: datetime = None, folder=1, **kwargs
     ) -> Union[AsyncIterator[Message], List[int]]:
-        """Yields messages received by the currently registered account.
+        """Yields messages received in the specified message box.
 
+        :param str message_box: the message box to get the messages from, can be obtained from get_message_boxes
         :param `datetime.datetime` last_sync: date of the last sync,
             gets only the objects updated since this date
         :param int folder: message folder: 1 - received; 2 - sent; 3 - deleted
         :rtype: Union[AsyncIterator[:class:`~vulcan.data.Message`], List[int]]
         """
-        return Message.get(self._api, last_sync, folder, **kwargs)
+        return Message.get(self._api, message_box, last_sync, folder, **kwargs)
+
+    async def get_messageboxes(
+        self, **kwargs
+    ) -> Union[AsyncIterator[Messagebox], List[int]]:
+        """Yields messageboxes.
+
+        :rtype: Union[AsyncIterator[:class:`~vulcan.data.Message`], List[int]]
```

Comment:
Return type is wrong. Also, wasn't `List[int]` only for returning deleted IDs? In case of message boxes, I think there aren't any deleted IDs returned. Correct me if I'm wrong; I didn't check the new APIs that much.